.gitignore

```diff
@@ -24,6 +24,16 @@ nbproject/private
 lib/cext/ruby.su
 lib/mri/openssl.su
 
+bench/chunky_png/oily_png/ext/oily_png/Makefile
+bench/chunky_png/oily_png/ext/oily_png/*.bc
+bench/chunky_png/oily_png/ext/oily_png/*.su
+bench/chunky_png/oily_png/lib/oily_png/oily_png.su
+
+bench/psd.rb/psd_native/ext/psd_native/Makefile
+bench/psd.rb/psd_native/ext/psd_native/*.bc
+bench/psd.rb/psd_native/ext/psd_native/*.su
+bench/psd.rb/psd_native/lib/psd_native/psd_native.su
+
```

Comment:
#255 has a checked-in Makefile for instance.
This helps to see if some `.su` or `.bc` got out of place, but we could ignore `*.bc` `*.su` globally.